packages/es-components/src/components/containers/notification/useNotification.js

```diff
@@ -125,6 +132,7 @@ export function useNotification(styleType = 'base') {
           variant={color}
           className={className}
           style={style}
+          bgColor={bgColor}
```

Comment:
This is the base component for LightNotification and Notification. We will add this property to set a specific background color